parm/config/gefs/config.fcst

```diff
@@ -119,24 +119,27 @@ if (( gwd_opt == 2 )); then
     #export  do_ugwp_v1_orog_only=".false."
 
     #--used for UFS p8
-    export knob_ugwp_version=0
+    export knob_ugwp_version=1
     export do_ugwp=".false."
     export do_tofd=".false."
-    export do_ugwp_v0=".true."
-    export do_ugwp_v1=".false."
+    export do_ugwp_v0=".false."
+    export do_ugwp_v1=".true."
     export do_ugwp_v0_orog_only=".false."
     export do_ugwp_v0_nst_only=".false."
-    export do_gsl_drag_ls_bl=".false."
+    export do_gsl_drag_ls_bl=".true."
     export do_gsl_drag_ss=".true."
-    export do_gsl_drag_tofd=".false."
+    export do_gsl_drag_tofd=".true."
     export do_ugwp_v1_orog_only=".false."
     launch_level=$(echo "${LEVS}/2.35" |bc)
     export launch_level
+    if [[ ${do_gsl_drag_ls_bl} == ".true." ]]; then
+       export cdmbgwd=${cdmbgwd_gsl}
+    fi
```

Comment:
It will not always be .true.  As I mentioned above, the user has flexibility to go back to the old GFSv16 orographic GWD if desired.  In that case, they would set 'do_ugwp_v0=.true.' and 'do_gsl_drag_ls_bl=.false.', then the 'old default' values of cdmbgwd would be needed.  The if-block is needed.

Reply:
I respectfully disagree.
The way it is currently set up, if `gwd_opt = 2`, then the statements on line 129 will set `do_gsl_drag_ls_bl=.true.`
In which case the if block is redundant.
However, I agree that if the user comes in and manually changes the default `do_gsl_drag_ls_bl=.false.` on line 129, then they will have to also have to comment out line 135 or 136.

Reply:
I agree that 'do_gsl_drag_ls_bl' is the default for 'gwd_opt=2', but it is not a requirement.  The 'gwd_opt=2' option originated with the 'unified_ugwp' CCPP_suite which allows use of either the 'old' GFS orographic GWD or the new 'GSL' scheme.  This switch is made with the 'do_gsl_drag_ls_bl' variable.  The proposed scheme for HR3 is the 'GSL' scheme, which is why setting it to .true. is the default.  But don't we want the flexibility for a user to revert back to the 'old' GFS scheme if desired?  This is what I had hoped for.